sw/host/opentitanlib/src/bootstrap/mod.rs

```diff
@@ -176,7 +174,7 @@ impl<'a> Bootstrap<'a> {
 
         if perform_bootstrap_reset {
             log::info!("Releasing bootstrap pins...");
-            self.bootstrap_pin.write(false)?;
```

Comment:
@cfrantz, not sure if this is possible but since the new protocol ends in reset, there is changes that the chip can re-enter the bootstrap mode before we release these pins. Maybe we should do this after the chip enters bootstrap, i.e. at the end of the first `if` block above.

Reply:
You could always re-apply a hardware reset after this.

It's not necessarily mandatory for the protocol to send the software reset sequence at the end.  The requirement is that the chip goes through some sort of reset after bootstrapping.

Reply:
Sure, my comment was based on the current implementation since IIUC this is shared by other bootstrap implementations. I'll resolve this since I didn't notice any issues, we can revisit this if needed.